src/utils/use-force-update.ts

```diff
@@ -6,10 +7,18 @@ export function useForceUpdate(): [VoidFunction, number] {
     const [forcedRenderCount, setForcedRenderCount] = useState(0)
     useUnmountEffect(() => (unloadingRef.current = true))
 
-    return [
-        useCallback(() => {
-            !unloadingRef.current && setForcedRenderCount(forcedRenderCount + 1)
-        }, [forcedRenderCount]),
-        forcedRenderCount,
-    ]
+    const forceRender = useCallback(() => {
+        !unloadingRef.current && setForcedRenderCount(forcedRenderCount + 1)
+    }, [forcedRenderCount])
+
+    /**
+     * Defer this to  the end of the next animation frame in case there are multiple
+     * synchronous calls.
+     */
+    const deferredForceRender = useCallback(
+        () => sync.postRender(forceRender),
+        [forceRender]
+    )
+
+    return [deferredForceRender, forcedRenderCount]
```

Comment:
What is the forcedRenderCount used for?

Reply:
It could, for instance, be used in cache breaking